packages/apps/plugins/plugin-status-bar/src/translations.ts

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { type useTranslation } from '@dxos/react-ui';
+
+import { STATUS_BAR_PLUGIN } from './meta';
+
+type StatusBarTranslationKeys =
+  | 'plugin name'
+  | 'feedback panel title'
+  | 'email input label'
+  | 'email input placeholder'
+  | 'feedback text area label'
+  | 'feedback text area placeholder'
+  | 'send feedback label'
+  | 'name label'
+  | 'name placeholder';
+
+type RequiredTranslation = { [key in StatusBarTranslationKeys]: string };
+type TranslationEntry = { [STATUS_BAR_PLUGIN]: RequiredTranslation };
+type Language = string;
+
+// -- helpers.
+const entry = (translation: RequiredTranslation): TranslationEntry => ({ [STATUS_BAR_PLUGIN]: translation });
+export const mkTranslation = (t: ReturnType<typeof useTranslation>['t']) => (key: StatusBarTranslationKeys) => t(key);
+
+const translations: Record<Language, TranslationEntry> = {
+  'en-US': entry({
+    'plugin name': 'Status Bar',
+    'feedback panel title': 'Feedback',
+    'name label': 'Name',
+    'name placeholder': 'Full name',
+    'email input label': 'Email',
+    'email input placeholder': 'your-email@example.com',
```

Comment:
placeholders should generally align in purpose — this is an example, whereas the name field is an expanded label… fwiw I’m not sure what the best approach is, really… maybe can just leave this alone.